src/libs/antares/study/parts/thermal/cluster_list.cpp

```diff
@@ -248,6 +265,16 @@ bool ThermalClusterLoadFromSection(const AnyString& filename,
 
     cluster.setName(section.name);
 
+    // initialize the ramping attributes only if ramping-enabled=true
+    auto* rampingEnabledProperty = section.find("ramping-enabled");
+    if(rampingEnabledProperty)
+    {
+        bool rampingEnabled = false;
+        bool attributeOK = rampingEnabledProperty->value.to<bool>(rampingEnabled);
+        if (rampingEnabled && attributeOK)
+            cluster.ramping = ThermalCluster::Ramping();
```

Comment:
```suggestion
            cluster.ramping.emplace();
```

Otherwise the temporary object is constructed and destructed immediately. In-place construction saves one Construction/Destruction, see https://en.cppreference.com/w/cpp/utility/optional/emplace